Assets/Scripts/ARFaceArkitBlendShapeVisualizer.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+using UnityEngine.XR.ARFoundation;
+using UnityEngine.XR.ARKit;
+
+/// <summary>
+/// Populates the action unit coefficients for an <see cref="ARFace"/>.
+/// </summary>
+/// <remarks>
+/// If this <c>GameObject</c> has a <c>SkinnedMeshRenderer</c>,
+/// this component will generate the blend shape coefficients from the underlying <c>ARFace</c>.
+/// 
+/// </remarks>
+[RequireComponent(typeof(ARFace))]
+public class ARFaceArkitBlendShapeVisualizer : MonoBehaviour
+{
+    [SerializeField]
+    float coefficientScale = 100.0f;

```

Comment:
Needs a public property to avoid warnings in the Editor.

Reply:
this does not produce a warning since there is a default value

Reply:
All `MonoBehaviour`s should be scriptable, which means you should have a public setter. Perhaps we can be more lax with sample code, but I don't see why we shouldn't just give this a public getter/setter.

Also, private fields follow a different naming convention from public properties:
```suggestion
    float m_CoefficientScale = 100.0f;
```